src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -762,13 +762,13 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
         } else if (var_annotation == "i32") {
             type = ASRUtils::TYPE(ASR::make_Integer_t(al, loc,
                 4, dims.p, dims.size()));
-        } else if (var_annotation == "i64") {
+        } else if (var_annotation == "i64" || var_annotation == "int") {
```

Comment:
This we should not do, since `int` means arbitrary precision integer, which we will eventually support in LPython as well, but that is not the same as `f64`.